app/Http/Controllers/Multiplayer/RoomsController.php

```diff
@@ -20,6 +20,13 @@ public function __construct()
         $this->middleware('require-scopes:public', ['only' => ['index', 'leaderboard', 'show']]);
     }
 
+    public function destroy($id)
+    {
+        $room = Room::findOrFail($id);
+        $room->endGame(\Auth()->user());
```

Comment:
it's a different thing 😔  `::user()`

Reply:
![1732026037](https://github.com/user-attachments/assets/5a83054b-4694-400b-89c0-8cdc9b35a957)

i don't even wanna know

Reply:
oh apparently i had the braces in there which i should not have? i give up

Reply:
it's a different way to access the same thing but slightly faster because the other method (`auth()`) does additional processing for some unknown reason